bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -335,7 +334,7 @@ func (r *KubeadmConfigReconciler) handleClusterNotInitialized(ctx context.Contex
 	err = certificates.LookupOrGenerate(
 		ctx,
 		r.Client,
-		types.NamespacedName{Name: scope.Cluster.Name, Namespace: scope.Cluster.Namespace},
+		client.ObjectKey{Name: scope.Cluster.Name, Namespace: scope.Cluster.Namespace},
```

Comment:
util.ObjectKey here?

Reply:
the scope of this PR is to standardize on client.ObjectKey and not yet on util.ObjectKey. I'll make an issue and we can talk about standardizing on util.ObjectKey later.

Reply:
actually looking at the rest of the PR i commented too early, my fault.